tests/resources/VCH-Util.robot

```diff
@@ -128,6 +128,54 @@ Get Docker Params
     Run Keyword If  ${port} == 2376  Set Environment Variable  VCH-PARAMS  -H ${dockerHost} --tls
     Run Keyword If  ${port} == 2375  Set Environment Variable  VCH-PARAMS  -H ${dockerHost}
 
+Get Compose Params  ${output}  ${certs}  ${dockerHost}
+
+Get Compose Params
```

Comment:
Loc has added some new keywords to the compose tests re: COMPOSE_PARAMS and certs. You can check out his PR and maybe reuse these after his PR gets merged